specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-04-01-preview/registries.json

```diff
@@ -507,56 +567,141 @@
       },
       "additionalProperties": false
     },
-    "Registry": {
-      "description": "Details of the Registry",
+    "PartialUserAssignedIdentity": {
       "type": "object",
-      "allOf": [
-        {
-          "$ref": "#/definitions/ResourceBase"
+      "additionalProperties": false
+    },
+    "PrivateEndpointConnectionARMResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "This is the private endpoint connection name created on SRP\r\nFull resource id: /subscriptions/{subId}/resourceGroups/{rgName}/providers/Microsoft.MachineLearningServices/{resourceType}/{resourceName}/privateEndpointConnections/{peConnectionName}",
+          "type": "string",
+          "x-nullable": true
+        },
+        "location": {
+          "description": "Same as workspace location.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "properties": {
+          "description": "Properties of the Private Endpoint Connection",
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties",
+          "x-nullable": true
         }
-      ],
+      },
+      "additionalProperties": false
+    },
+    "PrivateEndpointConnectionProperties": {
```

Comment:
>PrivateEndpointConnectionProperties [](http://example.com/codeflow?start=5&length=35)

Can you refactor and instead `$ref` to all the private endpoint connection typedefs from 

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/privatelinks.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/privatelinks.json)

so that you don't have to define them?

Reply:
PrivateEndpointConnectionProperties as defined here has "groupIds" property which is not listed in the definition you linked above.